src/parkapi_sources/converters/ellwangen/converter.py

```diff
@@ -0,0 +1,66 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from datetime import datetime, timezone
+from typing import Any
+
+from openpyxl.cell import Cell
+
+from parkapi_sources.converters.base_converter.push import NormalizedXlsxConverter
+from parkapi_sources.models import SourceInfo
+
+
+class EllwangenPushConverter(NormalizedXlsxConverter):
+
+    source_info = SourceInfo(
+        uid='ellwangen',
+        name='Ellwangen Parkdaten',
+        public_url='https://www.ellwangen.de',
+        has_realtime_data=False,
+    )
+
+    header_row: dict[str, str] = {
+        'ID': 'uid',
+        'Name': 'name',
+        'Art der Anlage': 'type',
+        'Betreiber Name': 'operator_name',
+        'Längengrad': 'lat',
+        'Breitengrad': 'lon',
+        'Adresse mit PLZ und Stadt': 'address',
+        'Maximale Parkdauer / min': 'max_stay',
```

Comment:
Is `/ min` part of the template now? Then I would suggest to changed it in `NormalizedXlsxConverter`.

Reply:
Hmm. To be clear about that: the normalized converter was meant as an template matching the template we provided at the start. Idea of that template was that it is not changed by cities. If you accept changed templates, this means no generic converter, but an adaption to every city who changes something. Would not recommend, but that's up to you :)

Reply:
Is it possible to define the ```header_row``` like this ```header_row: dict[str, str] = { 'Maximale Parkdauer / min': 'max_stay'}```. Such that, all other attributes from the ellwangen converter will be inherited from the ```header_row``` in the ```NormalizedXlsxConverter```. This way, i will not need to modify or change anything the ```header_row``` in the ```NormalizedXlsxConverter```

I tried it but i get the error ```AssertionError: There should be more valid then invalid parking sites```
Would that be possible to implement?

Reply:
`AssertionError: There should be more valid then invalid parking sites` - then you basically broke the whole converter, because that's actually what this test is for: to test of the converter gives a reasonable amount of results. So, this is by purpose, to prevent that any changes breaks the converter.

You can re-use a large dictionary from a parent class by extending it in your constructor. Then you don't need to re-define everything. Eg:

```
def __init__(...):
  self.header_row = {**super().header_row, 'new_key': 'new_value'}
```

Reply:
Okay I would just redefined the ```header_row``` as a whole